### ISSUE TITLE
Как связаться с автором проекта?

### ISSUE THREAD

**Issue by viciious** [MiGeRA/FlashKit-MD-Plus#5]
Здравствуйте!

Простите, что пишу не в тему и не туда, но не смог нигде найти email автора. Я один из разработчиков проекта Doom 32X Resurrection, и в нашем небольшом коммьюнити родилась идея обсудить с талантливыми людьми возможность создания open source дизайна RAM картриджа для Sega Genesis, чтобы можно было делать более интересные проекты для CD32X. Расскажите, интересно было бы Вам это в принципе, и как можно было бы с Вами связаться?

Спасибо!

**Comment by MiGeRA** [comment]
Интересно. Жаль что пока нахлабучки 32Х у меня в железе нет, но заиметь планирую (только не за конскую сумму денег, поэтому пока и нет, видимо).

А для привата и вложений больших всяких, полагаю, несложно догадаться о моей электронной почте на домене моего сайта, одноименной ему и моему нику также ;-)

**Comment by viciious** [comment]
Да, я уже после того как тикет на гитхабе создал, нашёл и адрес электропочты )

В целом идея создать какой-то более совершенный аналог уже ранее созданного 10 лет назад прототипа, но на более современных чипах:
![image](https://github.com/MiGeRA/FlashKit-MD-Plus/assets/1173058/06729a66-2326-4525-add7-91477782cc8c)

По поводу самого 32Х - у меня есть запасная PAL тушка, готов для пользы общего дела уступить за разумные деньги.

Если интересно, то давайте спишемся в почте.

**Comment by MiGeRA** [comment]
На фотке аля-RAM-картридж для Sega-CD? Только объемом в два мегабайта ... официальные вроде меньшего объема были. Если слот картриджем таким занимать, то код и все остальное только на CD хранить остается получается при таком решении? Или что-то не понял я?

Цель такого картриджа какая (в потребительском смысле)? Его ведь должен будет собрать (купить) потом пользователь решивший поиграть в игрушку его ресурс использующую, правильно? А игрушка на диске чтоль будет? Полагаю тогда и флэш-память вместе с оперативой нужно вешать на плату, имхо мало у кого есть "полная сега" и с 32X, и с оптическим приводом (а уж тем-более его железным эмулятором) ... А если так, то подобное решение, в принципе, существует уже лет 10 как (жаль повторить его нельзя, да и купить уже проблематично) - Mega Everdrive v2 (X7): в нем 16 мегабайт оперативы (в одной микросхеме). Из нее эмулируется ROM для запуска игр с карточки, но можно оставить ее и доступной для записи. Т.е. для отладки самое оно, а в "продакшен" уже можно оставить по минималке из реально востребованного.

Вобщем нужно определиться с целью данного картриджа на общем фоне, его ценой для конечного пользователя, легкостью к повторению (в т.ч. по доступности микросхем, их заменяемостью и простотой пайки - в Эвердрайве вон память в BGA-корпусе).

Про 32X: шнурок к ней для подключения к первому Мегадрайву есть? Разумная цена это сколько? - пишите в мыло ...
